tests/integration/benchmark_runner/benchmark_operator/test_oc_benchmark_operator.py

```diff
@@ -61,19 +61,19 @@ def before_after_each_test_fixture():
     # before all test: setup
     kinds = ('pod', 'vm')
     for kind in kinds:
-        __generate_yamls(workload='uperf', kind=kind)
+        __generate_yamls(workload='stressng', kind=kind)
```

Comment:
Ideally these would be parameterized, so you wouldn't have to change all this test code if you want to change what cases are run.

Reply:
Fixed